meinberlin/react/projects/ProjectTile.jsx

```diff
@@ -87,4 +88,32 @@ const ProjectTile = ({ project, isHorizontal, topicChoices, isMapTile }) => {
   )
 }
 
+function IconSwitcher ({ access }) {
```

Comment:
```suggestion
function ProjectTileIcon ({ access }) {
```
or something similar

Reply:
Thanks, that is now changed 👍